libraries/AP_Param/AP_Param.h

```diff
@@ -301,14 +302,26 @@ class AP_Param
     ///
     void notify() const;
     
-    /// Save the current value of the variable to EEPROM.
+    /// Save the current value of the variable to storage, synchronous API
     ///
     /// @param  force_save     If true then force save even if default
     ///
     /// @return                True if the variable was saved successfully.
     ///
-    bool save(bool force_save=false);
+    void save_sync(bool force_save=false);
 
+    /// flush all pending parameter saves
+    /// used on reboot
+    static void flush(void);
+    
+    /// Save the current value of the variable to storage
+    ///
+    /// @param  force_save     If true then force save even if default
+    ///
+    /// @return                True if the variable was saved successfully.
```

Comment:
I love comments.  This one needs fixing.

Reply:
fixed